chacractl/api/binaries.py

```diff
@@ -56,6 +57,20 @@ def sanitize_url(self, url_part):
             url = "%s/" % url
         return url
 
+    def load_file(self, filepath):
+        chsum = sha512()
+        binary = open(filepath, 'rb')
+        for chunk in iter(lambda: binary.read(4096), b''):
+            chsum.update(chunk)
+        binary.seek(0)
+        return binary, chsum.hexdigest()
+
+    def verify_upload(arch_url, filename, digest):
```

Comment:
can we change this so the negation reads better? Right now this is used like:

    if not self.verify_upload(...)

But that reads like if `verify_upload` is a flag or option. Something that can say that the upload was done correctly so that the negation can read better would be nice. For example `upload_is_verified`, `uploaded_ok`,  `upload_verified`,  or anything similar wouldn't look like an option when negating:

    if not upload_is_verified(...):